### ISSUE TITLE
Change to guiding mode

### ISSUE THREAD

**Issue by weristeddy** [TimSchneider42/franky#17]
Hello, I am wondering why i am not able to change the RobotMode?
I tried using the set_guiding_mode() function from the Robot class but my RobotMode after checking was still in Idle. Same also for the stop() function.

**Comment by weristeddy** [comment]
maybe good to know: i am using the Franka Emika Robot with libfranka 0.8.0

**Comment by TimSchneider42** [comment]
Did you press and hold the two buttons on the robots gripper?

**Comment by weristeddy** [comment]
yes, then a kind of purple light appears. Guiding mode some how only works if i press the black stop button before and then press the buttons for driving mode

**Comment by TimSchneider42** [comment]
Maybe you have to press the button to move it? I think the `set_guiding_mode` function only sets which dimensions can be guided in guiding mode.

**Comment by weristeddy** [comment]
okay, so there is no way to change the RobotMode from 'Idle' to either 'UserStopped' or 'Guiding' using only code? 
Are these modes only accessible when pressing manually the stop and gripper buttons?

**Comment by TimSchneider42** [comment]
To be honest, I am not a 100% sure how it works. I remember playing around with the function once and being able to lock rotation but not translation. I think I still had to press the stop button. Maybe it is best to ask in the Franka forum.

**Comment by weristeddy** [comment]
alright, thank you for your help :)
Do you maybe know what the robot.stop() function does?

**Comment by TimSchneider42** [comment]
Sorry for the late reply, I was on vacation. The stop function terminates the current motion and changes the robot's state back to idle. The userstopped mode is when the black button is pushed, I believe.
